internal/security/secretstoreclient/vault.go

```diff
@@ -46,15 +45,25 @@ func NewSecretStoreClient(logger logger.LoggingClient, r internal.HttpCaller, s
 	}
 }
 
-func (vc *vaultClient) HealthCheck() (statusCode int, err error) {
-	resp, err := vc.commonJSONRequest(nil, http.MethodGet, VaultHealthAPI, struct{}{})
-	if err != nil {
-		vc.logger.Error(fmt.Sprintf("failed on checking status of secret store: %s", err.Error()))
+func (vc *vaultClient) HealthCheck() (int, error) {
```

Comment:
All of the methods changed to use the common doRequest() method;
return arguments have been de-identifed to reduce risk of introducing unintended bugs.